src/js/shot-generator/Editor.js

```diff
@@ -666,6 +727,8 @@ const SceneManager = connect(
                 loaded: props.loaded ? props.loaded : false,
                 devices,
 
+                text: props.displayName,
```

Comment:
Probably don't need to pass in `text` anymore, as it's the same as `props.displayName`.

Reply:
removed all references to text